CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+Thanks for considering to contribute to the Modelica Standard Library.
+
+Contributing can be done by either filing an issue or by contributing code.
+
+When filing an issue, please include all information necessary to reproduce,
+and state what package is affected.
+
+The workflow for contributing code depends on whether you are a collaborator 
+with push rights, or an outside collaborator.
+
+The [workflow for outside collaborators](https://guides.github.com/activities/forking/) is usually as follows:  
+
+1. Fork the repository to your account by
+   [using the `Fork` button](https://help.github.com/articles/fork-a-repo/).
+2. Clone the forked repository to your computer.
+3. Create a new topic branch and give it a meaningful name, 
+   like e.g. `issue2161_contributing`.
```

Comment:
"e.g." is always written enclosed in commas, i.e., ",e.g.,". Same for "i.e." as you see ;-)